### PR TITLE
CSS Cascade

### PR DESCRIPTION
**Pull request by CollinHovey**
chrome_taAFMqydrI
chrome_5NfvralZOt
chrome_jggrlI4dmP

### CODE REVIEW

File: css-cascade/source-order.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.first-class {
```
Comment (Zhiwutian) [inline css-cascade/source-order.css:1]: you are missing one of the classes here.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

File: css-cascade/source-order.css (hunk 1 above)
Comment (CollinHovey) [reply css-cascade/source-order.css:1]: it's been done

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**